node/package.json

```diff
@@ -39,7 +39,7 @@
     "@types/minimatch": "3.0.3",
     "@types/mocha": "^9.1.1",
     "@types/mockery": "^1.4.29",
-    "@types/node": "^16.11.39",
+    "@types/node": "^20.3.1",
     "@types/q": "^1.5.4",
     "@types/semver": "^7.3.4",
     "@types/shelljs": "^0.8.8",
```

Comment:
How to ensure that the devDependencies in package.json are bumped up as needed? Node20 required typescript version upgrade, we update typescript version to latest 5.1.6 in node 20 tasks, so I put same version here and then run 'npm update' command.

Reply:
you could use some third-party tools like [npm-check](https://www.npmjs.com/package/npm-check) to validate that everything is fine